python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -1,9 +1,74 @@
+import json
+
+from django.views.generic import View
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render_to_response
 from django.template import RequestContext
 
+from django.contrib.gis.geos import GEOSGeometry, MultiPolygon
+from requests import get
+
+from destinations.models import Destination
+
 def home(request):
     print request.META['REMOTE_ADDR']
     return render_to_response('home.html', context_instance=RequestContext(request))
 
+
 def map(request):
     return render_to_response('map.html', context_instance=RequestContext(request))
+
+
+class Reachable(View):
```

Comment:
We could do both; they have different use cases. http://www.django-rest-framework.org/api-guide/views/#function-based-views

As an aside, I think the name of this class could be made a bit clearer; it's a bit generic if you don't know the context.